TTS/bin/train_glow_tts.py

```diff
@@ -468,7 +465,6 @@ def evaluate(data_loader, model, criterion, ap, global_step, epoch):
     return keep_avg.avg_values
 
 
-# FIXME: move args definition/parsing inside of main?
```

Comment:
This has to happen outside of main so that the experiment folder can be removed if it fails.